content/docs/authorization/deployment/_index.md

```diff
@@ -200,29 +200,67 @@ There are currently 2 ways of doing this:
 
 Given a setup where Kubernetes, a storage system, CSI driver(s), and CSM for Authorization are deployed, follow the steps below to configure the CSI Drivers to work with the Authorization sidecar:
```

Comment:
Change "Given a setup where Kubernetes, a storage system, CSI driver(s), and CSM for Authorization are deployed" should be changed to say ".., and the CSM for Authorization Proxy Server are deployed".

Reply:
fixed